pages/_app.js

```diff
@@ -82,6 +83,7 @@ export default function App({ Component, pageProps }) {
           onToggleFavorite={handleToggleFavorite}
         />
       </main>
+      <NavBar/>
```

Comment:
I would highly recommend you use the Layout Component from next here. This is the common way of implementing a Layout for every page on the app instead of here in the _app.js
Here is the [link to the docs ](https://nextjs.org/docs/pages/building-your-application/routing/pages-and-layouts#layout-pattern)

your _app.js would look cleaner like this:
```js
return (
   <>
      <GlobalStyle />
      <Layout>
        <Component
          {...pageProps}
         activityData={activityData}
          favoriteActivitiesList={favoriteActivitiesList}
          onAddActivity={handleAddActivity}
          onEditActivity={handleEditActivity}
          onDeleteActivity={handleDeleteActivity}
          onToggleFavorite={handleToggleFavorite}
        />
      </Layout>
  </>
)
```

Your Layout component would be located in your compoents folder under `Layout.js` and look something like this:

```js
import Header from "./Header";
import NavBar from "./NavBar";

export default function Layout({ children }) {
  return (
    <>
      <Header />
      <main>{children}</main>
      <NavBar />
    </>
  )
}
```